tools/wasi-headers/Cargo.toml

```diff
@@ -7,6 +7,6 @@ edition = "2018"
 publish = false
 
 [dependencies]
-clap = "2"
 heck = "0.3.1"
 witx = "0.5.0"
```

Comment:
A path dependency to witx is appropriate now that we're tracking WASI as a submodule, imo.

Reply:
Makes sense; done.